extensions/functions_rounding.yaml

```diff
@@ -0,0 +1,106 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "ceil"
+    description: "Rounding to the ceiling of the value"
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i8
```

Comment:
I don't know what rounding an i8 (or any of the i* values below) means.

Reply:
Arrow does not support integer round explicitly.  However, we will auto-cast an integer column to a double column and then call round on that.  I think this has led to some confusion in the docs.  I agree with @jacques-n that these kernels should not exist.